api/app/tests/apps/signals/test_filters_v1.py

```diff
@@ -0,0 +1,297 @@
+from datetime import datetime, timedelta
+
+from django.db import connection
+from rest_framework.test import APITestCase
+
+from signals.apps.signals.models import CategoryAssignment, Signal, Status
+from signals.apps.signals.workflow import BEHANDELING, ON_HOLD
+from tests.apps.signals.factories import (
+    LocationFactory,
+    PriorityFactory,
+    ReporterFactory,
+    SignalFactory,
+    StatusFactory,
+    SubCategoryFactory
+)
+from tests.apps.users.factories import SuperUserFactory
+
+
+class TestFilters(APITestCase):
+    """ Tests filters on the private v1 list endpoint: /signals/v1/private/signals """
+    SIGNALS_DISTRIBUTE_DAYS_CNT = 10
+    SIGNALS_DISTRIBUTE_HOURS_CNT = 10
+    SUBCATEGORY_CNT = 5
+    LIST_ENDPOINT = '/signals/v1/private/signals/'
+
+    signals = []
+    sub_categories = []
+
+    @classmethod
+    def _create_signals(cls):
+        cls.signals = [SignalFactory.create() for _ in
```

Comment:
See comment when updating the created and updated at. 

Also it is possible to let the factory create batches of signals.

The following will create 5 signals
```
SignalFactory.create_batch(5)
```

Also it is possible to overwrite values when creating signals with the factory.

The following will create a signal with the text "Lorem ipsum..."
```
SignalFactory.create(text='Lorem ipsum...')
```

Reply:
Good to know :)